book/website/project-design/sdpw/pd-sdp-private-learning.md

```diff
@@ -43,6 +43,21 @@ We can also draw on the experience of research in the field of cross-domain trai
 
 Differential privacy has also seen significant use as a technique for preserving privacy during model training, reducing the risk of the model learning individual data points too well by adding small amounts of statistical noise during training {cite:ps}`boulemtafesReviewPrivacypreservingTechniques2020,feyisetanPrivacyUtilitypreservingTextual2020`. 
 
+## Case study: Data privacy in ML neuroimaging
+### Why do we want to share neuroimages?
+Neuroimaging is an emerging medical technique focusing on the collection and analysis of brain-related data. Typical sub-fields are EEG, MEG, MRI, fMRI and PET. Neuroimaging methods commonly result in one large, longitudinal data set per individual, which in turn makes the method suitable for the application of machine learning algorithms. At the same time, ML techniques can be costly and time consuming (for example, the average scanning cost for 1hr of fMRI analysis is estimated to be [between USD 1.600-8.400](https://www.itnonline.com/content/mri-costs), with scanning times for an fMRI experiment lasting between 1.5 and 3hrs.)  These efforts associated with the collection of neuroimaging data have led to the foundation of large neuroimaging consortia, with the aim to create data sets that go beyond sample sizes of N=20-30 of a typical single neuroimaging study. Often, such consortia focus on a specific disorder or mental or cognitive state.
```

Comment:
Might be worth adding this definition of neuroimaging to the glossary: https://the-turing-way.netlify.app/afterword/glossary.html 😄

Reply:
Also, for: EEG, MEG, MRI, fMRI and PET... Maybe write "electroencephalogram (EEG), Magnetoencephalography (MEG)... etc) for people that don't know the terms :)

Reply:
Same applies to using machine learning (ML)!

Reply:
Removed acronyms and added a reference for details. I think mentioning these method in this chapter is not necessary.